### ISSUE TITLE
Failed to mount component

### ISSUE THREAD

**Issue by marlarius** [jofftiquez/vue-stripe-checkout#73]
Hi,

I keep getting this warning:

```
[Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <VueStripeCheckout>
       <HelloWorld> at src/components/HelloWorld.vue
         <App> at src/App.vue
           <Root>
```

And the component does not show up.

This is my component file:

```
<template>
<div>
  <h1>vue-stripe-checkout</h1>
  <vue-stripe-checkout
    ref="checkoutRef"
    :items="items"
    :successUrl="successUrl"
    :cancelUrl="cancelUrl"
  >
    <template slot="checkout-button">
      <button @click="checkout">Shutup and take my money!</button>
    </template>
  </vue-stripe-checkout>
</div>
</template>
 
 
<script>

import Vue from 'vue';
import VueStripeCheckout from 'vue-stripe-checkout';
Vue.use(VueStripeCheckout, {
  publishableKey: 'your-publishable-key'
});

export default {
  components:  {
    VueStripeCheckout
  },
  data: () => ({
    loading: false,
    items: [
      {
        sku: 'sku_FdQKocNoVzznpJ', 
        quantity: 1
      }
    ],
    successUrl: 'your-success-url',
    cancelUrl: 'your-cancel-url',
  }),
  methods: {
    checkout () {
      this.$refs.checkoutRef.redirectToCheckout();
    }
  }
}
</script> 
```

What could be wrong?

**Comment by jofftiquez** [comment]
@marlarius this must be caused by the wrong build type. Can you try to do this:

```
Vue.use(VueStripeCheckout.default, {
  publishableKey: 'your-publishable-key'
});
```

Add `.default` in the `VueStripeCheckout`.

**Comment by marlarius** [comment]
Now that gives me this error:

vue.runtime.esm.js?2b0e:5100 Uncaught TypeError: Cannot read property 'install' of undefined
    at Function.Vue.use (vue.runtime.esm.js?2b0e:5100)
    at eval (HelloWorld.vue?140d:22)
    at Module../node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/HelloWorld.vue?vue&type=script&lang=js& (app.js:831)
    at __webpack_require__ (app.js:727)
    at fn (app.js:101)
    at eval (HelloWorld.vue?9697:1)
    at Module../src/components/HelloWorld.vue?vue&type=script&lang=js& (app.js:2203)
    at __webpack_require__ (app.js:727)
    at fn (app.js:101)
    at eval (HelloWorld.vue?f177:1)

**Comment by jofftiquez** [comment]
@marlarius noted. You should remove .default. I will look into this, will fix for the next update (soon). Thank you for your feedback.

**Comment by marlarius** [comment]
FYI, I have attached the simple demo project, based on the standard "Hello World" vue app. Maybe that can tell you something.

Note that I have only included vue-stripe-checkout in node_modules.

[stripe.zip](https://github.com/jofftiquez/vue-stripe-checkout/files/3578645/stripe.zip)

**Comment by jofftiquez** [comment]
Thank you @marlarius

**Comment by jofftiquez** [comment]
I was able to reproduce the problem. Will fix this soon.

**Comment by jeremyd125** [comment]
Is there a fix for this yet?

**Comment by marlarius** [comment]
I need it badly too.

**Comment by jofftiquez** [comment]
Hello all, I merged the version 3 branch and made some fixes. Please upgrade and try the new version. Closing this now. Please create a new issue if there's any. Thank you.

IMPORTANT - there are breaking changes in the implementation, please read the documentation carefully.
